kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/JsonUtil.java

```diff
@@ -16,18 +16,56 @@
 
 package org.kitesdk.data.spi;
 
+import com.fasterxml.jackson.core.JsonFactory;
```

Comment:
Can we shade Jackson please to avoid all the compatibility problems we've seen with it.